pages/linux/turbostat.md

```diff
@@ -0,0 +1,24 @@
+# turbostat
+
+> Report processor frequency and idle statistics.
+> More information: <https://www.linux.org/docs/man8/turbostat.html>.
+
+- Display usage for the most common parameters:
```

Comment:
We generally use "Display help" in all pages for help commands, but we open some exceptions. Maybe here we can open one too.

```suggestion
- Display help for the most common parameters:
```
Or:
```suggestion
- Display help:
```